tuning_docs/tensile_tuning.tex

```diff
@@ -846,10 +941,10 @@ \subsection{Using the Tensile Library For Testing}
 
 \subsubsection{Example Usage}
 
-Once the library is generated, it can be used in rocBLAS by setting the environment variable ROCBLAS\_TENSILE\_LIBPATH. Execute rocblas-bench as follows:
+Once the library is generated, it can be used in rocBLAS by setting the environment variable rocBLAS\_TENSILE\_LIBPATH. Execute rocBLAS-bench as follows:
```

Comment:
```suggestion
Once the library is generated, it can be used in rocBLAS by setting the environment variable ROCBLAS\_TENSILE\_LIBPATH. Execute rocBLAS-bench as follows:
```